cola/__main__.py

```diff
@@ -0,0 +1,19 @@
+"""
+Run cola as a Python module.
+
+Usage:
+
+    python -m cola
+
+"""
```

Comment:
All of our source file start with the same import, and our pylint checks enforce it.  Please add this line immediately after the docstring:

    from __future__ import absolute_import, division, unicode_literals

Sorry about that.